content/articles/making-predictions-using-monte-carlo-simulation/index.md

```diff
@@ -0,0 +1,299 @@
+
+This means that, dependencies between the inputs should be represented.
+
+III. **Computate the inputs**
+
+The computation should be deterministic.
+
+By deterministic, we mean that they should be able to give the same output each time they are computed.
+
+IV. **Combine the results**
+
+Assemble the information to create the desired outcome.
+
+Typical outputs include:
+
+_Histogram_
+
+Statistics in a nutshell (mean, variance, and standard deviation)
+
+### Examples
+
+Now, let's have a look at several examples using python.
+
+For beginners, you can use any text editor with python extensions.
+
+You can check this article [here](https://re.sajari.com/token/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwdXJwb3NlIjoiY2xpY2siLCJkZXN0aW5hdGlvbiI6Imh0dHBzOi8vd3d3LnNlY3Rpb24uaW8vZW5naW5lZXJpbmctZWR1Y2F0aW9uL2EtYmVnaW5uZXJzLWd1aWRlLXRvLXB5dGhvbi8iLCJ2YWxzIjp7ImNvbGxlY3Rpb24iOlsic2VjdGlvbi1pbyJdLCJjb21wYW55IjpbIjE1NjgyMzQxMTYwNzg2MTQ4NjgiXSwiZmllbGQiOlsidXJsIl0sInByb2plY3QiOlsiMTU2ODIzNDExNjA3ODYxNDg2OCJdLCJxLmlkIjpbInQybGt6eHU0ZmV1ZGJmaXAiXSwicS5zbCI6WyI3Il0sInEudWlkIjpbInQybGt6eHU0ZmV1ZGJmaXAwIl0sInZhbHVlIjpbImh0dHBzOi8vd3d3LnNlY3Rpb24uaW8vZW5naW5lZXJpbmctZWR1Y2F0aW9uL2EtYmVnaW5uZXJzLWd1aWRlLXRvLXB5dGhvbi8iXX19.xc7CDwfGaO-RpZ6vlYGjoQLwxYJ2Fi_9oeA_f0owC0c) for more details on installations and some python basic functions.
+
+#### Example 1: Approximate the value of pi
+
+Consider a best-fitting circle that can be slotted into a square with a range of ℝ2 over _[−1, 1]²_.
+
+• the circle radius = 1 and area = π.
+
+• the square has an area of _2² = 4_.
+
+• the ratio between their areas is thus _𝜋/4_.
+
+**Solution**
+
+The following Monte Carlo approach can be used to approximate the value of π:
+
+1. Draw the square over _[−1, 1]²_.
+
+2. Draw the circle with the longest radius inside the square.
+
+3. Scatter a large number P of grains over the square.
+
+4. Count the grains that fell in the circle.
+
+5. The grains counted divided by P and multiplied by 4 is an approximation.
+
+Using the four main steps, the pi can be estimated as follows:
+
+In this example, we have implemented the use of **Numpy library** in python.
+
+If are not familiar with python libraries, you can refer to this [article](https://re.sajari.com/token/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJwdXJwb3NlIjoiY2xpY2siLCJkZXN0aW5hdGlvbiI6Imh0dHBzOi8vd3d3LnNlY3Rpb24uaW8vZW5naW5lZXJpbmctZWR1Y2F0aW9uL2ludHJvZHVjdGlvbi10by1udW1weS8iLCJ2YWxzIjp7ImNvbGxlY3Rpb24iOlsic2VjdGlvbi1pbyJdLCJjb21wYW55IjpbIjE1NjgyMzQxMTYwNzg2MTQ4NjgiXSwiZmllbGQiOlsidXJsIl0sInByb2plY3QiOlsiMTU2ODIzNDExNjA3ODYxNDg2OCJdLCJxLmlkIjpbInZ0MHNzMGNnMHpiaTBjeG4iXSwicS5zbCI6WyIxIl0sInEudWlkIjpbInZ0MHNzMGNnMHpiaTBjeG4wIl0sInZhbHVlIjpbImh0dHBzOi8vd3d3LnNlY3Rpb24uaW8vZW5naW5lZXJpbmctZWR1Y2F0aW9uL2ludHJvZHVjdGlvbi10by1udW1weS8iXX19.1ya9PWvNq5r63W06KrKimy2-_BlOEOncpfNvLxpd648) for more understanding.
+
+##### **Define possible inputs**
```

Comment:
are these parts supposed to be among the  **basic steps** or **examples**? 🤔

Reply:
They are showing the implementation of the basic steps in solving a problem.